src/configuration/configuration.ts

```diff
@@ -14,6 +14,10 @@ export interface IHandleKeys {
   [key: string]: boolean;
 }
 
+export interface IStatusBarColors {
```

Comment:
For super mega bonus points you could use TypeScript's mapped types to ENSURE that every mode has a color by mapping from our ModeNames object. Sort of like this:

`interface IStatusBarColors { [Key in keyof ModeNames]: string }`

Mapped types are awesome.